reaktive/src/commonMain/kotlin/com/badoo/reaktive/disposable/DisposableWrapper.kt

```diff
@@ -1,46 +1,21 @@
 package com.badoo.reaktive.disposable
 
-import com.badoo.reaktive.utils.atomic.AtomicReference
-import com.badoo.reaktive.utils.atomic.getAndUpdate
-
 /**
  * Thread-safe container of one [Disposable]
  */
-class DisposableWrapper : Disposable {
-
-    private val ref = AtomicReference<Holder?>(Holder(null))
-    override val isDisposed: Boolean get() = ref.value == null
+@Suppress("EmptyDefaultConstructor")
```

Comment:
Quite strange, that we need to suppress it.
https://github.com/arturbosch/detekt/blob/master/detekt-rules/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/empty/EmptyDefaultConstructor.kt#L30

Reply:
Can you try `expect class DisposableWrapper expect construction()`?

Reply:
`expect` is not allowed for constructors :-(

Reply:
Try just constructor then?

Reply:
Still fails